LEGACY_SPECIFICATION.md

```diff
@@ -1,4 +1,4 @@
-> ⚠️ **OUT OF DATE** - Current protocol description and specification has moved to the [Ceramic docs portal](https://developers.ceramic.network/protocol/)
+> ⚠️ **OUT OF DATE** - Current protocol description and specification has moved to the [Ceramic docs portal](https://developers.ceramic.network/docs/introduction/intro)
```

Comment:
@JustinaPetr Is there any chance we can unbreak this on our docs instead? I've been using /protocol in a few different places.

Reply:
I can add a redirect for this which should fix the issue here